dozer/cogs/ftc.py

```diff
@@ -230,6 +271,19 @@ async def team(self, ctx: DozerContext, team_num: int):
                         value=', '.join((team_data['city'], team_data['stateProv'], team_data['country'])) or "Unknown")
             e.add_field(name='Org/Sponsors', value=team_data.get('nameFull', "").strip() or "_ _")
             e.add_field(name='Website', value=website or 'n/a')
+            e.add_field(name='Team Info Page', value=f'https://ftcscout.org/teams/{team_num}')
+
+            if sres.status != 404:
+                team_stats = await sres.json(content_type=None)
+                e.add_field(name='Total OPR', value=str(round(team_stats['tot']['value'], 1)) + ", rank #" + str(
```

Comment:
NIT: Would strongly prefer this to be a fstring to avoid the string type conversions and explicit rounding.
```python
value=f"{team_stats['tot']['value']:.0}, rank #{team_stats['tot']['rank']}"
```
(the `:.0` at the end of the first format string specifies a [format specifier](https://peps.python.org/pep-0498/#format-specifiers) for zero printed values after the decimal point)

Reply:
Should be resolved in [`fe02dfb` (#460)](https://github.com/FRCDiscord/Dozer/pull/460/commits/fe02dfb44dc8b81d3958446b75ae45c775aea5e7)